### PR TITLE
Use streams instead of direct video urls

### PR DESCRIPTION
**Pull request by sindrig**
Over the last couple of months, the addon has become unusable for me for everything other than the live stream - video starts for maybe 5-10 seconds and then stops. 

By using the same strategy used by the new player on ruv.is and fetching an m3u8 catalog, the startup slows down a bit but watching the video works again.

### CODE REVIEW

**Comment by sindrig** [comment]
I also did some linting here and there... Hope that's ok!

File: plugin.video.sarpur/sarpur/actions.py [hunk 1]
```diff
@@ -51,21 +53,34 @@ def play_url(url, name):
         player.play(video_url, name)
 
 
-def play_video(file, name):
+def play_video(serie_episode, name):
```
Comment (sindrig) [inline plugin.video.sarpur/sarpur/actions.py:56]: This is essentially the biggest change.

**Comment by Dagur** [comment]
Fantastic! I'll do a little testing and then ask Kodi to update the plugin 👍

**Comment by Dagur** [comment]
I've created a pull request at Kodi. You can follow it here https://github.com/xbmc/repo-plugins/pull/2002

**Comment by sindrig** [comment]
Thanks!

Really nice response time :)

**Pull request merged**